origin.spec

```diff
@@ -244,6 +244,7 @@ do
   echo "+++ INSTALLING ${bin}"
   install -p -m 755 _output/local/bin/${PLATFORM}/${bin} %{buildroot}%{_bindir}/${bin}
 done
+# setcap 'cap_net_bind_service=ep' %{buildroot}%{_bindir}/openshift
```

Comment:
Not sure where to place setcap inside the RPM spec and it also requires root.

cc: @smarterclayton @tdawson @stevekuznetsov